help_contextual_popup/README.rst

```diff
@@ -0,0 +1,35 @@
+Add contextual help popup
+=========================
+
+This module add a contextual html help popup on each model action.
+You also may use dedicated field custom_help on actions to add specific help
+
+Installation
+============
+
+It was tested on Odoo 8.0 branch.
+
+Credits
+=======
+
+Akretion
```

Comment:
Remove the company from here (it is intended for individuals). You already put the company of authors list.